samples/ktor-docker-sample/build.gradle.kts

```diff
@@ -20,5 +20,11 @@ ktor {
                 password = providers.environmentVariable("DOCKER_HUB_PASSWORD")
             )
         )
+        environmentVariables.add(
```

Comment:
Could we also have `environmentVariable("name", "value")` extension?

Reply:
Thanks for the feedback.  I've added an environmentVariable function.